tests/cucumber/features/authentication/authentication.feature

```diff
@@ -7,3 +6,13 @@ Feature: Login
   Background:
     Given I am on the home page
     Given I am signed out
+
+  @dev
+  Scenario: Create account
+    Given "test@test.com" has not registered
```

Comment:
That's because `meteor mongo` is not running on the mirror, it's running on the standard meteor instance.